ee/clickhouse/views/test/test_clickhouse_retention.py

```diff
@@ -40,45 +151,65 @@ def test_can_get_line_chart_and_fetch_people(self):
                 date_from="2020-01-01",
                 total_intervals=2,
                 date_to="2020-01-02",
-                display="ActionsLineGraph",
                 period="Day",
                 retention_type="retention_first_time",
+                breakdowns=[Breakdown(type="event", property="os")],
             ),
         )
 
-        trend_series = retention["result"][0]
-        retention_by_day = get_people_for_retention_trend_series(client=self.client, trend_series=trend_series)
+        retention_by_cohort_by_period = get_by_cohort_by_period_from_response(response=retention)
 
-        assert retention_by_day == {
-            "2020-01-01": ["person that leaves on 2020-01-02", "person that stays forever"],
-            "2020-01-02": ["person that stays forever"],
+        assert retention_by_cohort_by_period == {
+            "Chrome": {"1": 1, "2": 1},
+            "Safari": {"1": 1, "2": 1},  # IMPORTANT: the "2" value is from past the requested `date_to`
         }
```

Comment:
`self.assertEqual` would be more informative in case of a mismatch

Reply:
`pytest` handles informative output for this, would you say this is not as good as self.assertEqual? I can change if not.

Reply:
This gives some examples of the assert pytest introspection: https://docs.pytest.org/en/latest/example/reportingdemo.html

Reply:
Ohh, didn't know Pytest does this. Comes down to just code style then. Still would propose `assertEqual` in general for uniformity, but not important

Reply:
Yes, don't want to distract from the main issue here so have updated to use `self.assertEqual`